src/test/java/com/sap/cloud/yaas/service_sdk/example/java_jersey_wishlist/api/AbstractResourceTest.java

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Duplicate of [AbstractResourceTest.java](https://github.com/SAP/yaas_java_jersey_wishlist/blob/master/src/test/java/com/sap/wishlist/api/generated/AbstractResourceTest.java)?